releasenotes/notes/add-linear-slack-dqm-25312a5fe23574e6.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add `linear` option as a new `slack_method` for
```

Comment:
Need ` `` ` rather than ``` ` ``` for the code snippets in RST

Reply:
this is a science :), I fixed that.

Reply:
You missed the ones around `DiscreteQuadraticModel.add_linear_inequality_constraint`